src/database.ts

```diff
@@ -1308,6 +1402,60 @@ class Database extends GrpcServiceObject {
       callback!(err, transaction);
     });
   }
+
+  /**
+   * Query object for listing database operations.
+   *
+   * @typedef {object} GetDatabaseOperationsRequest
+   * @property {boolean} [autoPaginate=true] Have pagination handled
+   *     automatically.
+   * @property {number} [maxApiCalls] Maximum number of API calls to make.
+   * @property {number} [maxResults] Maximum number of items to return.
+   * @property {number} [pageSize] Maximum number of results per page.
+   * @property {string} [pageToken] A previously-returned page token
+   *     representing part of the larger set of results to view.
+   */
+  /**
+   * @typedef {array} GetDatabaseOperationsResponse
+   * @property {IOperation[]} 0 Array of {@link IOperation} instances.
+   * @property {object} 1 The full API response.
+   */
+  /**
+   * List pending and completed operations for the database.
+   *
+   * @see {@link Instance.getDatabaseOperations}
+   *
+   * @param {GetDatabaseOperationsOptions} [options] Contains query object for
```

Comment:
This needs a typedef to appear in docs.

Reply:
This was a bit tricky because I realized that we don't use GetDatabaseOperationsRequest anymore. So I removed all those typedefs. Also means that we no longer support maxApiCalls and maxResults but I suppose those are CallOptions. Note that this is different to how `Instance#getDatabases()` is configured.

Reply:
This kind of hooks into [this discussion](https://github.com/googleapis/nodejs-spanner/pull/855#discussion_r409599808), in terms of if and how we should handle previously/currently-incorrectly accepted options.